core/internal/cltest/cltest.go

```diff
@@ -165,6 +165,10 @@ func MustRandomBytes(t *testing.T, l int) (b []byte) {
 	return b
 }
 
+func FormatWithPrefixedChainID(id, chainID string) string {
+	return fmt.Sprintf("%s/%s", chainID, id)
```

Comment:
This is kind of hard to read, and the opposite order could be surprising.
WDYT about inlining these cases? I think it will be less characters than this func name, and you can drop some explicit `.String()` calls too.

Reply:
Flipped the params, but not sure about inline. I feel like this stands out more and is less likely to make someone scratch their head if some test results needs to be prefixed. Though this is very simple so may not be an issue idk